integ-test/src/test/java/org/opensearch/sql/sql/WildcardQueryIT.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.sql;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_BEER;
+
+import java.io.IOException;
+import org.json.JSONObject;
+import org.junit.Test;
+import org.opensearch.sql.legacy.SQLIntegTestCase;
+
+public class WildcardQueryIT extends SQLIntegTestCase {
+
+  @Override
+  protected void init() throws Exception {
+    loadIndex(Index.BEER);
+  }
+
+  @Test
+  public void test_wildcard_query_function() throws IOException {
+    String query1 = "SELECT * FROM " + TEST_INDEX_BEER + " WHERE wildcard_query(Tags, 't*')";
+    JSONObject result1 = executeJdbcRequest(query1);
+    assertEquals(10, result1.getInt("total"));
+
+    String query2 = "SELECT * FROM " + TEST_INDEX_BEER + " WHERE wildcardquery(Tags, 't*')";
+    JSONObject result2 = executeJdbcRequest(query2);
+    assertEquals(10, result2.getInt("total"));
+
+    assertEquals(result1.getInt("total"), result2.getInt("total"));
+  }
+
+  @Test
+  public void test_wildcard_query_sql_wildcard_conversion() throws IOException {
```

Comment:
Maybe we should split this up into two tests.

Reply:
Definitely.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/commit/a2ca906329ab194b845b8078eafeae48eb4bf1c9